airflow/kubernetes/kube_client.py

```diff
@@ -123,5 +128,8 @@ def get_kube_client(
     if conf.getboolean('kubernetes', 'enable_tcp_keepalive', fallback=False):
         _enable_tcp_keepalive()
 
+    if not conf.getboolean('kubernetes', 'verify_ssl', fallback=True):
+        _disable_verify_ssl()
+
     client_conf = _get_kube_config(in_cluster, cluster_context, config_file)
     return _get_client_with_patched_configuration(client_conf)
```

Comment:
```suggestion
    if not conf.getboolean('kubernetes', 'verify_ssl', fallback=True):
        configuration.verify_ssl = False

    client_conf = _get_kube_config(in_cluster, cluster_context, config_file)
    if conf.getboolean('kubernetes', 'verify_ssl', fallback=True):
        
    return _get_client_with_patched_configuration(client_conf)
```
It seems to me that in this way we will avoid modifying the global configuration.

Reply:
@mik-laj this way seems is invalid for incluster mode